modules/cluster-autoscaler-machineset.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying-autoscaling.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="cluster-autoscaler-machineset{context}"]
+= Configuring the machineset
+
+The MachineSet which creates the Windows Machines requires configuring a file template. On creation, a `windows-user-data` secret is created. On subsequent use, the `windows-user-data` may be used by the MachineSets. The associated machinesets may have the following labels:
+
+* `machine.openshift.io/os-id: Windows`
+* `machine.openshift.io/cluster-api-machine-role: worker`
+* `machine.openshift.io/cluster-api-machine-type: worker`
+
+The following label has to be added to the Machine spec within the MachineSet spec:
+
+* `node-role.kubernetes.io/worker:``
++
+[IMPORTANT]
+====
+If these labels are not added, the the Windows node will not be marked as a `worker` node.
+====
+
+.Procedure
+
+. Configure the machineset template with values from your environment:
```

Comment:
This example is the vSphere template. I would like to combine Azure, vSphere, and AWS into one example if that is possible.